File: Api-20190322T003006Z-001/Api/TesteUpload/TesteUpload/ClientApp/src/models/usuario-model.ts
import { RifasModel } from './rifas-model';

export class UsuarioModel {
    id: any;
    usuario: string;
    senha: string;
    nome: string;
    telefone: number;
    rifa: RifasModel;
    codigoRifa: number;
    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}

File: Api-20190322T003006Z-001/Api/TesteUpload/TesteUpload/Model/UsuarioModel.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace TesteUpload.Model
{
    [Table("Usuario")]
    public class UsuarioModel
    {
        public UsuarioModel()
        {
        }

        [Key]
        public int Id { get; set; }
        [Required]
        public string Nome { get; set; }
        [Required]
        public string Telefone { get; set; }
        [Required]
        public string Estado { get; set; }

        public bool? Ganhador  { get; set; }
        public int CodigoRifa { get; set; }
        public DateTime dataOperacao { get; set; }
        public bool? Ativo { get; set; }
        public int IdRifa { get; set; }
        [ForeignKey("IdRifa")]
        public virtual RifaModel Rifa { get; set; }

    }
}